app/models/podcast_ownership.rb

```diff
@@ -0,0 +1,6 @@
+class PodcastOwnership < ApplicationRecord
+  belongs_to :owner, class_name: "User"
+  belongs_to :podcast
+  validates :podcast_id, uniqueness: { scope: :owner_id }
+  validates :podcast_id, :owner_id, :role, presence: true
```

Comment:
I believe this has been mutuated from https://github.com/forem/forem/pull/11354/files#diff-80ed95ceccb910e831a5c371a28a8b35b37729d2d4ad1d063702c9a084086b1fR4-R5 but this model doesn't have the `role` column, and it shouldn't have.

The uniqueness validation if needed needs to be backed by a DB index, see https://github.com/forem/forem/pull/11354/files#diff-1953bb445767352adbf509787151f41c6f293305f5b4d65189ccb33a15a261c3R11

ps. the name of the column is `user_id`, `:owner` it's just an alias to help us humans in the application logic :), if you check [belongs_to](https://api.rubyonrails.org/classes/ActiveRecord/Associations/ClassMethods.html#method-i-belongs_to) options section you can find how `belongs_to :owner, class_name: "User"` works